sdk/python/feast/client.py

```diff
@@ -353,6 +360,145 @@ def archive_project(self, project):
         if self._project == project:
             self._project = FEAST_DEFAULT_OPTIONS[CONFIG_PROJECT_KEY]
 
+    def apply_entity(
+        self, entities: Union[List[EntityV2], EntityV2], project: str = None
+    ):
+        """
+        Idempotently registers entities with Feast Core. Either a single
+        entity or a list can be provided.
```

Comment:
What if we just make this a single entity? Do we need to support a list?

Reply:
I think it's better to support a list, especially in the case if we only allow users to register entity through yaml config. This may require us to have a new yaml file for every new entity that's to be registered. Unless we're okay with that.